### PR TITLE
Redo ACT label handling

### PR DESCRIPTION
**Pull request by JoryHogeveen**
See https://github.com/pods-framework/pods/issues/3736 and https://github.com/pods-framework/pods/issues/3735

### CODE REVIEW

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
There are also a lot of labels within the UI templates. What shall we do with those?

Plus, (for the i18n component). Since there is nog "register" filter for ACT's, how about adding one so the labels can be filtered?

File: classes/PodsAdmin.php [hunk 1]
```diff
@@ -1038,26 +1037,88 @@ public function admin_setup_edit_options ( $pod ) {
                     'default' => '',
                     'object_type' => array( 'taxonomy' )
                 ),
-                // @todo Why was label_view added previously? Can't find it in WP
```
Comment (JoryHogeveen) [inline classes/PodsAdmin.php]: This was a ACT label only

File: classes/PodsAdmin.php [hunk 2]
```diff
@@ -1017,13 +1017,12 @@ public function admin_setup_edit_options ( $pod ) {
                     'default' => '',
                     'object_type' => array( 'taxonomy' )
                 ),
-                // @todo Why was label_edit added previously? Can't find it in WP
```
Comment (JoryHogeveen) [inline classes/PodsAdmin.php]: This was a ACT label only

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
There are 3 locations I placed a filter. I'd say it would be better to do this at an earlier stage, but I'm not sure if caching is possible for languages that aren't done with Polylang or WPML.

`pods_advanced_content_type_pod_data` and `pods_advanced_content_type_pod_data_**POD_NAME**`

Please let me know if you would like to keep it as it is now or if you want to relocate this filters to an earlier stage. Also let me know where, I was thinking about `PodsMeta` right after loading the Pod objects.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
Could you reply on my last comment?

> Please let me know if you would like to keep it as it is now or if you want to relocate this filters to an earlier stage. Also let me know where, I was thinking about PodsMeta right after loading the Pod objects.

**Comment by sc0ttkclark** [comment]
@JoryHogeveen i don't think the filter like `pods_advanced_content_type_pod_data` is going to be the right way to go, we should wait for the magic of PodsObject in 2.8 to enable something like that.

**Comment by JoryHogeveen** [comment]
In that case please make a reminder or something :)

**Pull request merged**